lib/custom-logger.js

```diff
@@ -83,7 +83,7 @@ function logRequestCompleted(req, reply, next) {
       response: {
         statusCode,
         body: {
-          bytes: reply.getHeader('content-length'),
+          bytes: parseInt(reply.getHeader('content-length')),
```

Comment:
What if content-length has an invalid number? I think there will be a NaN in our log, how does this get serialised? Can you add a test?